canonical-websites/www.ubuntu.com
# Created an rfp page with contextual footers

## Done

- created an RFP contact form with contextual footers
- cleaned up the copy doc
- didn't add it to the navigation.yaml for now
- updated the version of Django per Robin request
- looks like prettier cleaned up some sass

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/rfp](http://0.0.0.0:8001/rfp)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1OA_Eg1200fHpt7YYkE4qubiWfxL_U5fnlA0OObRe2n8/edit#heading=h.l28gk4yojgm6)
- Test the form
- See that it is linked from: http://0.0.0.0:8001/openstack/storage, http://0.0.0.0:8001/openstack/features and http://0.0.0.0:8001/openstack/consulting


## Issue / Card

Fixes #4082
